test/test_multiprocessing.py

```diff
@@ -939,4 +941,8 @@ def test_is_shared_cuda(self):
 
 
 if __name__ == '__main__':
-    run_tests()
+    # multiprocessing ipc use case not supported on Jetson
+    if IS_JETSON:
+        warnings.warn("skipping multiprocessing tests for Jetson since multiprocessing ipc use case is not supported")
```

Comment:
Why do you want to skip CPU tests in this case? Perhaps one one should be doing is to set `TEST_CUDA_IPC` to false if `IS_JETSON` is True?

Reply:
Good point, let me add that